test/conformance/cts_exe.py

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env python3
+"""
+ Copyright (C) 2023 Intel Corporation
+
+ Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+ See LICENSE.TXT
+ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+"""
+
+import sys
+from argparse import ArgumentParser
+import subprocess  # nosec B404
+
+if __name__ == '__main__':
+
+    parser = ArgumentParser()
+    parser.add_argument("--test_command", help="Ctest test case")
+
+    args = parser.parse_args()
+    result = subprocess.Popen([args.test_command, '--gtest_brief=1'], stdout = subprocess.PIPE, text = True)  # nosec B603
```

Comment:
`subprocess.run` ?

Reply:
Popen is a better version of `run`, I use it here on purpose because as a result, I get the TextIOWrapper class on which I can perform additional operations that are not possible on the str object in the case of  `run`.